config/routes.rb

```diff
@@ -3,13 +3,25 @@
   root to: 'projects#index'
 
   namespace :api do
-    resources :deploys, only: [:index, :active_count] do
+    resources :deploys, only: [:active_count] do
       collection do
         get :active_count
       end
     end
 
-    resources :stages, only: [:index] do
+    resources :projects, param: :project_id, only: [] do
+      get :duplicable, on: :member, to: 'stages#duplicable'
```

Comment:
hmm yeah works too ... saw `member do` more often ...

On Fri, Aug 12, 2016 at 3:58 PM, Adam Panzer <notifications@github.com>
wrote:

> In config/routes.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r74671898>:
>
> >        collection do
> >          get :active_count
> >        end
> >      end
> >
> > -    resources :stages, only: [:index] do
> > +    resources :projects, param: :project_id, only: [] do
> > +      get :duplicable, on: :member, to: 'stages#duplicable'
>
> What's wrong with on: :member ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/122035a3f8635be3fd046f6620704db40db5d7c0#r74671898>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ-MUyGoEzYZUVGjUm6uo0abS6Rzpks5qfPqYgaJpZM4Jhtog>
> .
>